requirements.txt

```diff
@@ -1,2 +1,2 @@
 click
-click-datetime
\ No newline at end of file
+click-datetime
```

Comment:
ok, I'm not familiar with it, I was just testing and it crashed due to that missing module so I added it to the list.

Reply:
Ok, that's fine, it can be cleaned out later as deemed necessary. I did a cursory read-through and I think it's best if we merge this now. I've been working to simplify the API and tests, so I hope I'll get that done sooner than later and we will then prepare a new release.

Thanks a lot to you and @newAM for working on this, it's much appreciated and awaited feature! :-)